hypervisor/qemu/qemu.go

```diff
@@ -27,6 +27,25 @@ type QemuContext struct {
 	qmpSockName string
 	cpus        int
 	process     *os.Process
+	interfaces	[]lazyQemuNicConfig
+	images		[]lazyQemuImageConfig
+	callbacks	[]hypervisor.VmEvent
+}
+
+//qemu lazy nic configuration 
+type lazyQemuNicConfig struct {
```

Comment:
could you remove the lazy prefix, use qemuNicConfig.
the "lazy" will be rename.